result/dhcpclientaddresses.yaml

```diff
@@ -0,0 +1,85 @@
+components:
+  schemas:
+    Dhcpv4ClientAddresses.State.Request:
+      description: >-
+        The request to retrieve DHCP Client learned address information.
+      type: object
+      properties:
+        dhcp_client_names:
+          description: >-
+            TBD
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Dhcpv4client/properties/name"
+          x-field-uid: 1     
+    Dhcpv4ClientAddresses.State:
+      description: >-
+       Addresses learned by DHCP clients.
+      type: object
+      properties:
+        dhcp_client_name:
+          description: >-
+            The name of a DHCP Client.
+          type: string
+          x-field-uid: 1
+        ipv4_addresses:
+          type: array
+          items:
+            $ref: '#/components/schemas/DhcpClientv4.State'
+          x-field-uid: 2
+    DhcpClientv4.State:
+      description: >-
+        IPv4 unicast prefix.
+      type: object
+      properties:
+        ipv4_address:
+          description: >-
+            The IP address associated with this DHCP Client session.
+          type: string
+          x-field-uid: 1
+        prefix_length:
+          description: >-
+            The length of the prefix.
+          type: integer
+          format: uint32
+          maximum: 32
+          x-field-uid: 2
+        gateway_address:
+          description: >-
+            The Gateway address associated with this DHCP Client session.
+          type: string
+          x-field-uid: 3
+        lease_time:
+          description: >-
+            The duration of the IP address lease, in seconds.
+          type: string
+          x-field-uid: 4
+    DhcpClientv6.State:
+      description: >-
+        IPv6 unicast prefix.
+      type: object
+      properties:
+        ipv6_address:
+          description: >-
+            The IP address associated with this DHCP Client session.
+          type: string
+          x-field-uid: 1
+        prefix_length:
+          description: >-
+            The length of the prefix.
+          type: integer
+          format: uint32
+          maximum: 128
+          x-field-uid: 2
+        gateway_address:
+          description: >-
+            The Gateway address associated with this DHCP Client session.
```

Comment:
How will we know that this is not sent by server and e.g. using configured gw instead.